wikimedia/wikipedia-ios
# Add empty view to watchlist feature

**Phabricator:** https://phabricator.wikimedia.org/T342552

### Notes
* This PR adds the empty view for the watchlist feature
* On the filter empty view, only some of the subtitle is a button. That was because we can't use attributed strings in SwittUI yet, so I had to break down the sentence and create a small button in the middle.

### Test Steps
1. On an account with no watchlist items, make sure you see the empty view, and it redirects to the app's search
2. Add some watchlist items, and make sure you see them. Then activate all the filters to see the filter option for the empty view, and that it redirects to the filter view.
